ci-bench/src/main.rs

```diff
@@ -77,6 +78,11 @@ pub enum Command {
     },
     /// Run a single benchmark at the provided index (used by the bench runner to start each benchmark in its own process)
     RunSingle { index: u32, side: Side },
+    /// Run all benchmarks in walltime mode, each one for 100 times, and print the measured timings in CSV format
```

Comment:
"each one for 100 times" - I think this part of the comment might have fallen out of sync with the implementation, it's `iterations_per_scenario` times each right? Or have I misunderstood?

Reply:
Sorry, the comment fell indeed out of sync! Thanks for pointing it out